exonum/exonum
# Polish docs

This PR edits several doc comments for clarity. A specific problem that it solves is the misleading naming of constants in the example for the `message!` macro, but it goes much deeper.

I will continue adding commits in this PR, but I would like for some version of it to be included in the upcoming 0.5 release.

### What kind of change does this PR introduce?

Other (docs fixes).

### Checklist:

- [x] All commits are named based on our guideline (read CONTRIBUTING.md)
- [ ] ~Tests for the changes have been added~ (not applicable)
- [x] Docs have been added / updated
- [ ] CHANGELOG.md have been updated

### How can we test these changes?

Fetch and checkout the PR locally and build the docs using `cargo doc --no-deps`. Then see the docs corresponding to the introduced changes.
